### ISSUE TITLE
Read-only mode for accounts with 0 balance

### ISSUE THREAD

**Issue by kcole16** [near/near-wallet#1131]
### Overview
To use the wallet right now, users need to have enough NEAR in their account to cover the cost of adding a new access key. 

We should allow users to access their accounts any time they have a full access key or 2FA, regardless of their account balance. This includes viewing recovery methods, which currently requires a signature from the newly created localStorage key, but which should instead use the recovery key to authenticate.

### Stories
As a user of the wallet with an available balance of 0, I want to view my asset portfolio and staking positions, so I can know the value of my account.

As a user of the wallet with an available balance of 0, I want to view my enabled recovery options, so I can understand the ways in which my account is accessible.

**Comment by vgrichina** [comment]
> This includes viewing recovery methods, which currently requires a signature from the newly created localStorage key, but which should instead use the recovery key to authenticate.

We shouldn't do this. This somewhat breaks 2FA security, which depends on the fact that recovery key is not normally available on user's machine. 

We can change how the connection to contract helper is made however, solving https://github.com/near/near-wallet/issues/772 in process as well. Basically if we store this connection in contract helper itself – we won't need to pay for gas to add new "viewing" key. However we still need to add new local key for signing transactions (vs using recovery key directly).

**Comment by vgrichina** [comment]
Maybe we should ask user to fund account during recovery (similar to funding during creation) if there isn't enough tokens?

**Comment by kcole16** [comment]
@vgrichina Fair enough re: viewing recovery methods. At the very least, the user should be able to view their account at all times (especially since we've made `/profile` public).

We can also ask them to fund their account, but it should be while in some sort of read-only mode.

**Comment by vgrichina** [comment]
@kcole16 very low key "solution" I guess can be just to redirect them to /profile if there is not enough funds to recover?
